src/main/webapp/app/overview/course-overview.component.ts

```diff
@@ -201,6 +210,19 @@ export class CourseOverviewComponent implements OnInit, OnDestroy, AfterViewInit
         }
     }
 
+    loadLearningGoals() {
+        forkJoin([this.learningGoalService.getAllForCourse(this.courseId), this.learningGoalService.getAllPrerequisitesForCourse(this.courseId)]).subscribe({
```

Comment:
Is there any reason why we don't get the prerequisites and the learning goals in the same call?

Reply:
That would require another endpoint on the server and a DTO class.